### ISSUE TITLE
Cluttered Advanced Tooltips

### ISSUE THREAD

**Issue by Revvilo** [scalda/Foolcraft_3#254]
## FEATURE REQUEST
- [ ] I would like to propose an additional mod.
- [x] I would like to propose a config change.
- [ ] I would like to propose a custom recipe.
- [ ] I would like to propose another change.

## FEATURE INFORMATION

I noticed that the tooltips are quite cluttered with duplicate information while **advanced tooltips** are enabled (F3 + h). So I did a little research and found that Immersive Engineering, and D³ Core are both currently adding Ore Dict tooltips.

So I suggest that we could;
- Disable `oreToolips` from Immersive Engineering
- Disable `enableStringID` from D³ Core
- & possibly disable `enableUnlocalizedName` (also from D³ Core)

in order to clean up the tooltips a bit while just hovering over them.

Now if you wanted to go all the way (I just thought of this), the mod called EnderCore also adds a *hold shift* version of the Ore Dict tooltips, and funnily enough is already enabled. So you could actually disable ALL normal display of the Ore Dict (by disabling the rest of D³ Core's tooltip settings) and leave it to only show when shift is pressed. (This is done with EnderCore's `showOredictTooltips` cfg option being set to `2`)

It's small things like these that you don't really notice and might not be a big issue, but I feel like it just makes it that bit more polished. Let me know what you think :)

## Visual representation

| Current | Tweaked |
| --- | --- |
| <img src="https://user-images.githubusercontent.com/13816212/43364475-e3c91024-936e-11e8-8481-796539354693.png"> | <img src="https://user-images.githubusercontent.com/13816212/43364480-ec9fbafe-936e-11e8-8061-297893f2b372.png"> |

### Shift only

| Without Shift held | With Shift held |
| --- | --- |
| <img src="https://user-images.githubusercontent.com/13816212/43364620-eaa98862-9371-11e8-980c-fd1a87e74ef8.png"> | <img src="https://user-images.githubusercontent.com/13816212/43364616-e0f576b4-9371-11e8-8116-84f3456ee219.png"> |

**Comment by scalda** [comment]
Will fix for 1.5.1 thanks for this

**Comment by Revvilo** [comment]
Oh, real quick - EnderCore has an option enabled that shows the item's ID when F3 - H is enabled. Effectively duplicating the vanilla feature that already exists.

_endercore.cfg_
`I:showRegistryNameTooltips=3`

I had already disabled that before I made this issue and didn't realise that it's enabled in the pack or by mod default.

<img src="https://user-images.githubusercontent.com/13816212/44063661-a0fb0290-9fb5-11e8-84a1-ac3518a68b78.png" width="300px">

Just thought I'd mention this, too, in case you wanted to disable that also. 👍
